### ISSUE TITLE
about train model issues!

### ISSUE THREAD

**Issue by CvHadesSun** [zhaogaofeng611/TextMatch#2]
Hi,I want to know how to define the train data (validation data) label?now i am define 1 means match ,and 0 means not match,but I train model ,there is a error occurs,I find maybe the label doesn't match to network output logits and probs .Thanks!

**Comment by CvHadesSun** [comment]
oh sorry !I train RE2 model!

**Comment by CvHadesSun** [comment]
oh sorry ! i am training the RE2 model ,so can you give me some advice for above issues?
